examples/next/pages/ui/components/authenticator/sign-in-sms-mfa/index.page.tsx

```diff
@@ -10,6 +11,18 @@ Amplify.configure({
   },
 });
 
+I18n.putVocabularies(dict);
```

Comment:
For now, yes. Because this might look different in React vs Angular vs Vue. Suggestion proposed by Eric on my last PR - https://github.com/aws-amplify/amplify-ui/pull/189#discussion_r690727346